content/blog/lifehacking/2017-12-01_freedom_and_productivity.md

```diff
@@ -0,0 +1,199 @@
+a tool. And it seems that people forget what is their own motivation in the
+first place, what is their *why*. Why do you want to be more productive or
+successful anyways? Do you want to work less? Or to spend more time with your
+friends and family? Or to spend more time on hobbies and passions that can make
+you feel more fulfilled? Or is your motivation entirely different, and you want
+to raise up in your company's hierarchy, by rising up your throughput, the
+quantity or quality of your work? All motives are valable here of course. But
+there should be one in the first place, otherwise motivation might not quite
+follow, and thus the practice of productivity routines will be aborted very
+soon after they get started. This is often a very strong sign for myself, when
+I consider doing something, add it to a todo list and months later it is still
+there; maybe it wasn't that useful or valuable, and I can erase it with a light
+heart.
+
+And when the initial, kickstarting motivation is right there, the *why* should
+also help set a limit to the endless quest of productivity. To know when to
+stop. For instance, when you managed to get more time for a dedicated hobby,
+and you're plain satifised, what would be the point of squeezing even more in
+your limited free time? Remember, productivity is a means to an end, not an end
+by itself. Without this in mind, you could be chasing an endless goal.
+
+People get easily fooled into thinking that more working hours imply more
+quality work. That's a societal feeling: in a capitalistic society, being
+productive means contributing to growth thus the good economic health of a
+entity greater than yourself (your country, your region, the world). And since
+people mix up being productive with working long hours, the confusion is easy
+to make. But I think productivity is the opposite of this: trying to work the
+least number of hours to advance work on the most important and valuable
+things. Begin maximally efficient with the minimum amount of resources.
+Identify those actions which will have the most positive impact, help the
+greatest number of people, be *valuable*. So it is another time to acknowledge
+the need for a why: why do you work for, in the first place? Is the work you're
+doing valuable, or are you doing busy work (like handling emails, answering
+very quickly on instant chat systems, etc.) to make it look like you're
+productive? Or more humbly, one might just be working to get the regular pay
+check, since work doesn't have to be a source of fulfillment, but instead can
+just be a means to an end too. Again, all motives are good here; but there
+should be one to make sense out of it.
+
+## To do what you want, to want what you do
+
+I think that's one goal of productivity: to be more conscious of all the
+choices we're making (what to work on) so that our work is more in adequation
+with our principles and real objectives. Thus it is about freedom, that is, the
```

Comment:
Remark: If you have a job that you dislike, just because you need the money and it's hard to see better alternatives, this may hurt a little.

Reply:
Well, as he mentioned earlier in this post, productivity is a topic for rich people anyway... :P

Reply:
That's a good point; I'll add a sentence explaining that work is not only what we do in a job, but also activities on our free time.